src/lammpsio/dump.py

```diff
@@ -1,13 +1,19 @@
 import gzip
 import os
 import pathlib
+import sys
 
 import numpy
 
 from .box import Box
 from .data import _readline
 from .snapshot import Snapshot
 
+try:
+    import pyzstd
+except ModuleNotFoundError:
+    pass
+
```

Comment:
I prefer to use the pattern of setting a variable when the import works or fails, which we can then reuse below.
```suggestion
try:
    import pyzstd
    _has_pyzstd = True
except ModuleNotFoundError:
    _has_pyzstd = False

```